.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -6,12 +6,12 @@ about: Suggest an idea for this project
 
 <!--
 Please report issues regarding specific projects in their respective issue trackers, e.g.:
- - Akka HTTP: https://github.com/akka/akka-http/issues
- - Alpakka:   https://github.com/akka/alpakka/issues
- - Akka Persistence Cassandra Plugin: https://github.com/akka/akka-persistence-cassandra/issues
+ - Pekko: https://github.com/apache/incubator-pekko/issues
```

Comment:
Could you replace (in 2 templates):
```
Pekko: https://github.com/apache/incubator-pekko/issues
```
with
```
Pekko Connectors: https://github.com/apache/incubator-pekko-connectors/issues
```

Pekko Connectors is the equivalent of Alpakka in the old template